src/SamplesApp/SamplesApp.macOS/SamplesApp.macOS.csproj

```diff
@@ -123,6 +123,10 @@
     <ProjectReference Include="..\..\Uno.UI.FluentTheme\Uno.UI.FluentTheme.csproj">
       <Project>{535339e2-6c01-4478-9889-f6e76b1c00f4}</Project>
       <Name>Uno.UI.FluentTheme</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\SamplesApp.UITests.Generator\Uno.Samples.UITest.Generator.csproj">

```

Comment:
This should not be needed, it is included through another file. It will also create an invalid dependency because the generator is not supposed to be bundled with the app.

Reply:
Without this (referencing the project) I was still getting the build error because the Generator was not being built.

Reply:
Project Reference removed! :)